Resources/Prototypes/Entities/Structures/Holographic/projections.yml

```diff
@@ -0,0 +1,30 @@
+- type: entity
+  id: HolosignWetFloor
+  name: wet floor sign
+  description: The words flicker as if they mean nothing.
+  components:
+  - type: Physics
+    bodyType: Static
+  - type: Sprite
+    sprite: Structures/Holo/wetfloor.rsi
+    state: icon
+  - type: InvisibleWall

```

Comment:
This component seems like bad code to me. Please rename it to something like "DeleteAfterTime" and move it off of mime because such a specific purpose component for something general is bad.

Reply:
I changed this in my mime PR so when that gets merged it can be changed.